Sources/TeslaSwift/Model/Authentication.swift

```diff
@@ -7,7 +7,6 @@
 //
 
 import Foundation
-import CryptoKit
```

Comment:
Can you revert all changes here as the current Master branch already has the auth working correctly?